config/default.yml

```diff
@@ -999,210 +999,10 @@ RSpec/Capybara:
   Include: *1
   Language: *2
 
-RSpec/Capybara/CurrentPathExpectation:
-  Description: Checks that no expectations are set on Capybara's `current_path`.
-  Enabled: true
-  VersionAdded: '1.18'
-  VersionChanged: '2.0'
-  Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/Capybara/CurrentPathExpectation
-
 RSpec/Capybara/FeatureMethods:
```

Comment:
Can we replace featuremethods with RSpec/Dialect configuration?
Would close a bunch of issues, too:
![Uploading 01ADFB5F-F2D7-400B-AD48-8D4AEF039F11.jpeg…]()

Reply:
Good idea @pirj. I’ve added this as [an issue in the 3.0 milestone](https://github.com/rubocop/rubocop-rspec/issues/1852).